examples/ats-keyword/.pre-commit-config.yaml

```diff
@@ -11,12 +21,8 @@ repos:
     rev: v13.0.0
     hooks:
     -   id: clang-format
-        exclude: >
-            (?x)^(
-                bsp/aws_configs/|
-                bsp/aws_libraries/|
-                bsp/freertos-platform/|
-                bsp/default_credentials/|
-                bsp/tf_m_targets/|
-                kws/ota/
-            )
+-   repo: https://git.gitlab.oss.arm.com/engineering/iot-m-sw/open-iot-sdk/tools/developer-tools.git
```

Comment:
should there be a second line commented out, that points to the public gitlab repo, then a comment line explaining what to do?  I guess we could just document that in the CI section of the documentation.